Functions/Environment.Tests.ps1

```diff
@@ -106,10 +103,10 @@ InModuleScope -ModuleName Pester {
             Mock 'GetPesterOs' {
```

Comment:
The name of the test still contains the old path. "returns '/tmp' directory for MacOS"

Reply:
Will fix